docs/docs/components/service-mesh.md

```diff
@@ -14,6 +14,17 @@ all workloads which have a specified configuration. The service mesh container f
 sets up `iptables` rules based on its configuration and then starts
 [Envoy](https://www.envoyproxy.io/) for TLS origination and termination.
 
+## Service mesh startup enforcement
+Since Contrast doesn't yet enforce the order in which the containers are started
+(see [Limitations](../features-limitations.md)), we deny all incoming connections
+until the service mesh is fully configured.
+A systemd unit inside the podVM creates this deny rule.
+The kata-agent systemd unit requires that this unit successfully runs and exists.
+Therefore, the deny rule is in place before any containers can be started.
+
+If the user specifies no service mesh annotations, the Initializer takes care
+of removing the deny rule.
```

Comment:
I just realized that this is a mildly breaking change, since you can't run containers without an initializer anymore (typical debug workflow for me). This is acceptable, but should be pointed out in release notes, I suppose.

Reply:
Note that the containers will still start but any readiness probe/incoming network will fail. But for your debug workflow you should be able to manually remove them via execing into the container. Though changing you debug container to just delete the rule should be easiest.

Reply:
Yeah, it's niche enough and workarounds exist.